core/client/src/main/java/alluxio/client/file/options/OpenFileOptions.java

```diff
@@ -91,6 +120,16 @@ public OpenFileOptions setLocationPolicy(FileWriteLocationPolicy locationPolicy)
   }
 
   /**
+   * @param policy the block location policy for the UFS read
+   * @return the UFS read location policy
+   */
+  @JsonIgnore
```

Comment:
correct, but to be consistent with the file location policy, you should add the method `setBlockLocationPolicyClass` so that REST API clients can specify the BlockLocationPolicy class using the class name